ros_ign_gazebo/launch/ign_gazebo.launch.py

```diff
@@ -30,9 +30,14 @@ def generate_launch_description():
     return LaunchDescription([
         DeclareLaunchArgument('ign_args', default_value='',
                               description='Arguments to be passed to Ignition Gazebo'),
+        # The default version for Foxy in Citadel (3)
+        DeclareLaunchArgument('ign_version', default_value='3',
+                              description='Ignition version'),
```

Comment:
I think it's worth explaining a bit more here. Is this `citadel`? How about:

```suggestion
                              description='Ignition Gazebo's major version (i.e. "3" for Citadel)'),
```